Effects/FlxFloodFill/source/FlxFloodFill.hx

```diff
@@ -0,0 +1,131 @@
+/**
+ * 
+ * A HaxeFlixel port of Photonstorm's 'FloodFillFX':
+ * https://github.com/photonstorm/Flixel-Power-Tools/blob/master/src/org/flixel/plugin/photonstorm/FX/FloodFillFX.as
+ * 
+ */
+
+package;
+
+import flash.display.BitmapData;
+import flash.geom.Point;
+import flash.geom.Rectangle;
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+
+/**
+ * "Creates a flood fill effect FlxSprite, useful for bringing in images in cool ways"
+ */
+class FlxFloodFill extends FlxSprite
+{
+	private var complete:Bool = false;
+	private var isFilling:Bool = false;
```

Comment:
Couldn't `FlxBasic`'s `active` property be used instead (also making `start()` and `stop()` unnecessary)?